src/libraries/System.Globalization/tests/System/Globalization/RegionInfoTests.cs

```diff
@@ -63,6 +66,19 @@ public void CurrentRegion()
             }
         }
 
+        [ConditionalFact(nameof(RegionInfoPropertyTests.IcuIsNotLoaded))]
+        public void TestCurrentRegion()
+        {
+            RegionInfo ri = RegionInfo.CurrentRegion;
+            CultureInfo.CurrentCulture.ClearCachedData(); // clear the current region cached data
```

Comment:
Might this interact with other tests in any negative way? If memory serves, ClearCachedData mutates statics even though it's an instance method.

Reply:
Yes, it mutates static values but if something else fails in other tests, that means we have a problem in the test or in the product.

Reply:
Can you elaborate? What happens if, for example, this test runs concurrently with https://github.com/dotnet/runtime/blob/4f9ae42d861fcb4be2fcd5d3d55d5f227d30e723/src/libraries/System.Globalization/tests/CultureInfo/CultureInfoAll.cs#L635? Might one of these fail because of the unexpected interaction? I'm wondering if this and any other tests might need to be isolated from each other.

Reply:
It is possible the test you pointed at can fail. That test should run remotely as it is validating the cache itself. Any test code calling ClearCachedData should run fine except the test that validating the cache itself. I can change that test to execute remotely.

Reply:
I have changed the tests to execute remotely to avoid any possibility of failing.